site/src/Librecores/ProjectRepoBundle/Resources/views/Default/delete.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}Delete project - LibreCores{% endblock %}
+
+{% block content %}
+<h1>Project Deleted</h1>
+
+<p>Your project {{ Fqname }} has been deleted from LibreCores.<br><br>
```

Comment:
How about two <p></p> blocks instead of manual newlines created by <br>?

Reply:
use two paragraph blocks
&lt;p&gt;...&lt;/p&gt;